src/main/java/decodes/tsdb/compedit/PyFuncSelectDialog.java

```diff
@@ -77,31 +50,24 @@ public PyFuncSelectDialog(JFrame theFrame, PyFuncList funcList)
 		}
 	}
 
-	void guiInit() throws Exception
-	{
+	void guiInit() {
 		JPanel thePanel = new JPanel(new BorderLayout());
 		JPanel buttonPanel = new JPanel(new FlowLayout(FlowLayout.CENTER, 25, 8));
-		insertButton.addActionListener(new ActionListener()
-		{
-			public void actionPerformed(ActionEvent e)
-			{
-				insertPressed();
-			}
-		});
+		insertButton.addActionListener(e -> insertPressed());
 		buttonPanel.add(insertButton);
-		cancelButton.addActionListener(new java.awt.event.ActionListener()
-		{
-			public void actionPerformed(ActionEvent e)
-			{
-				cancelPressed();
-			}
-		});
+		cancelButton.addActionListener(e -> cancelPressed());
 		buttonPanel.add(cancelButton);
 
-		// Set up the table
-		if (model == null)
-			model = new FuncListTableModel(funcList);
-		tab = new SortingListTable(model, new int[] {30,70});
+		model = new DefaultTableModel(new String[]{"Name", "Signature","Function"},0);
```

Comment:
Since we're making this change anyways we should put these labels in the resource bundle: https://github.com/opendcs/opendcs/blob/master/src/main/resources/decodes/resources/compedit_en_US.properties

For future internationalization.

Reply:
done in latest commit